### PR TITLE
Remove 2 year option 177628351

### PR DESCRIPTION
**Pull request by lrotschy**
I poked around a bit. It looks like the constants I removed the two-year option from are only used on the HazardLegend component. The image below is from a throwaway branch off of fix-api-calls with this branch merged in: 
image

### CODE REVIEW

File: src/components/ui/map/legend/component.js [hunk 1]
```diff
@@ -10,6 +10,7 @@ import {
 
 // styles
 import './styles.scss';
+import { RETURN_PERIOD_OPTIONS } from 'components/hazard/map/legend/constants';
```
Comment (lbraun) [inline src/components/ui/map/legend/component.js]: Why do we need to import these now when we didn't before?

**Review by lbraun — APPROVED** [review]
@lrotschy I changed the base to gr-develop because I thought that was our base branch. Am I wrong?

It made the diff a lot cleaner in any case, so it seems as though that was the one you branched off of, right?

I'm not sure I totally understand your comment about the screenshot above. Maybe let's discuss on our check in today.

File: src/components/ui/map/legend/component.js (hunk 1 above)
Comment (lrotschy) [reply src/components/ui/map/legend/component.js]: Beats me.

**Comment by lbraun** [comment]
@lrotschy to remove new constant and then merge this into gr-develop, build the app and deploy to pantheon.

**Pull request merged**